libs/dh/market-participant/shell/src/lib/dh-market-participant-shell.module.ts

```diff
@@ -33,22 +34,22 @@ import {
   dhMarketParticipantOrganizationsCreatePath,
   dhMarketParticipantOrganizationsEditPath,
   dhMarketParticipantOrganizationsPath,
+  dhMarketParticipanGridAreaPath,
 } from '@energinet-datahub/dh/market-participant/routing';
 import { DhMarketParticipantEditActorComponent } from '@energinet-datahub/dh/market-participant/edit-actor';
+import {
+  DhMarketParticipantGridAreaScam,
+  DhMarketParticipantGridAreaComponent,
+} from '@energinet-datahub/dh/market-participant/feature-gridarea';
 
 const routes: Routes = [
-  {
-    path: '',
-    pathMatch: 'full',
-    component: DhMarketParticipantOrganizationComponent,
```

Comment:
Since a user should never end-up on that specific url, as they should always be on a specific page, i'd say this is acceptable behaviour, as they would only end up there if they manually edit the url.